package.json

```diff
@@ -4,12 +4,16 @@
   "description": "Curbside Restaurant Market place",
   "main": "server.js",
   "scripts": {
-    "start": " node server.js",
```

Comment:
If you remove this script, heroku wont start the backend unless heroku settings are changed. Are you sure this is what you want?